### PR TITLE
Add Google Analytics 4 script

### PR DESCRIPTION
**Pull request by atilafb**
- [x] Add Google Analytics 4 script
- [x] New env for Google Analytics

### CODE REVIEW

**Review by geeksilva97 — APPROVED** [review]

**Comment by carloslopes** [comment]
Nice work! But we can only merge this feature after we have the LGPD warning banner. Let's keep this on hold while we develop the banner.

File: .env.sample
Comment (talyssonoc) [inline .env.sample:31]: It would be nice to declare this variable here as well as non-required: https://github.com/Codeminer42/cm42-central/blob/master/app.json#L78

**Comment by geeksilva97** [comment]
> Nice work! But we can only merge this feature after we have the LGPD warning banner. Let's keep this on hold while we develop the banner.

Is there an ISSUE? I think @atilafb can take care of implementing this banner. 

cc @matheusazzi

**Comment by EduardoSCosta** [comment]
> 

I'm working on the LGPD banner already.

**Comment by EduardoSCosta** [comment]
> Nice work! But we can only merge this feature after we have the LGPD warning banner. Let's keep this on hold while we develop the banner.

[LGPD banner](https://github.com/Codeminer42/cm42-central/pull/792)

**Pull request merged**
